deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -127,8 +129,8 @@ async def get_info_of_uploads(
 
             return uploads
 
-    except Exception as e:
```

Comment:
Why do we not log the error anymore?

Reply:
We do, see https://docs.python.org/3/library/logging.html#logging.Logger.exception